app/backend/src/couchers/errors.py

```diff
@@ -34,6 +34,9 @@
 CANT_WRITE_REFERENCE_FOR_REQUEST = "You can't write a reference for that host request, or it wasn't found."
 CANT_WRITE_REFERENCE_INDICATED_DIDNT_MEETUP = "You can't write a reference for that host request because you indicated that you didn't meet up."  # fmt: skip
 CHAT_NOT_FOUND = "Couldn't find that chat."
+CHAT_INITIATION_THRESHOLD = (
+    "You have messaged a lot of users today. To avoid spam, you can't contact any more users for today."
```

Comment:
"Today" is ambiguous, use "24 hours" somehow

Reply:
I set a string representation of the interval constant as another constant and am using it everywhere now. This lets us adjust it flexibly.